SabreTools.FileTypes/BaseFile.cs

```diff
@@ -467,7 +487,8 @@ private static bool HasValidArchiveExtension(string path)
                 case "tlz":
                 case "zip":
                 case "zipx":
-
+                case "iso":
```

Comment:
This is dangerous to have enabled by default. Most people using SabreTools are used to having external hashes for ISOs. This essentially makes it an archive by default.